bitbots_ipm/launch/ipm.launch

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<launch>
+    <arg name="sim" default="false"/>
+    <arg name="full_image" default="false"/>
+    <arg name="markers" default="false"/>
+    <arg name="rviz" default="false"/>
+
+    <node pkg="ipm_image_node" exec="ipm" name="ipm_line_mask" output="screen">
+        <remap to="/camera/camera_info" from="camera_info"/>
+        <remap to="/line_mask_in_image" from="input"/>
+        <remap to="/line_mask_relative_pc" from="projected_point_cloud"/>
+        <param name="type" value="mask"/>
+        <param name="scale" value="0.2"/>
+        <param name="output_frame" value="base_footprint"/>
+        <param name="use_sim_time" value="$(var sim)"/>
+    </node>
+    <node pkg="soccer_ipm" exec="ipm" name="soccer_ipm" output="screen">
+        <remap to="/camera/camera_info" from="camera_info"/>
+        <param from="$(find-pkg-share bitbots_ipm)/config/soccer_ipm.yaml" />
+        <param name="use_sim_time" value="$(var sim)"/>
+    </node>
+
+    <group if="$(var full_image)">
+        <node pkg="ipm_image_node" exec="ipm" name="ipm_image" output="screen">
+            <remap to="/camera/camera_info" from="camera_info"/>
+            <remap to="/camera/image_proc" from="input"/>
+            <remap to="/projected_camera_image" from="projected_point_cloud"/>
+            <param name="type" value="rgb_image"/>
+            <param name="scale" value="1.0"/>
```

Comment:
This is only for debug purposes, and therefore we accept the performance hit by using scale=1?
Also params to config?

Reply:
Exactly.

This is the same as above. I find it simpler to define a parameter in the launch file if there are only a couple of them. Otherwise, one needs to look which config file is loaded, open this file etc.. This makes sense if you use many parameters that would clutter the launch file, but for a few it is unnecessary complexity imo..